packages/application-generic/src/usecases/subscriber-job-bound/subscriber-job-bound.usecase.ts

```diff
@@ -92,6 +93,7 @@ export class SubscriberJobBound {
         _organization: command.organizationId,
         channels: template?.steps.map((step) => step.template?.type),
         source: command.payload.__source || 'api',
+        subscriberSource: _subscriberSource || 'undefined',
```

Comment:
What is the case for the `subscriberSource` being undefined?

The value of `undefined` doesn't currently align with the enum value options. If we really need a falsy value, I think it's better to be more declarative with `null`.

Reply:
This should never happen at the current logic, but in the future, you never know so I added it in the backoff, null makes sense to me :)